ui/app/AppLayouts/Profile/panels/ProfileShowcaseAccountsPanel.qml

```diff
@@ -30,4 +30,13 @@ ProfileShowcasePanel {
         icon.color: model && model.colorId ? Utils.getColorForId(model.colorId) : Theme.palette.primaryColor3
         highlighted: model ? model.address === root.currentWallet : false
     }
+    filter: FastExpressionFilter {
+        expression: {
+            root.searcherText
+            return (address.toLowerCase().includes(root.searcherText.toLowerCase()) ||
```

Comment:
You could extract the `searcherText.toLowerCase()` expression somewhere in order not to re-evaluate it multiple times (same for other files)